src/lib/useImpliedTicket.js

```diff
@@ -0,0 +1,14 @@
+import * as ob from 'urbit-ob';
+
+import { prependSig } from './transformers';
+
+/**
+ * pull the suggested ticket from the #hash in the url
+ */
+export default function useImpliedTicket() {
```

Comment:
did we delete the version of this hook that I'd written at some point?

Reply:
I found and used an existing one for the implied point from the subdomain, if that's what you're thinking of?

Reply:
I thought I deleted that comment — sorry I'm slow today — I was just confused because of the similarities but I realized that they were different (and then didn't delete my confusing comment) — please ignore lol

Reply:
Pretty sure today's not a bridge day for you anyway, so you are more than forgiven. (;